### PR TITLE
poll protocol parameters from backend

### PR DESCRIPTION
**Pull request by yushih**

### CODE REVIEW

**Comment by neuodev** [comment]
[autoscirpt] Jira Ticket: **[[Q4] Dynamic protocol params](https://emurgo.atlassian.net/browse/YOEXT-855)**

[](__JIRA_COMMENT_FLAG)

File: packages/yoroi-extension/app/api/ada/index.js [hunk 1]
```diff
@@ -583,6 +593,12 @@ export const DEFAULT_ADDRESSES_PER_PAPER = 1;
 
 export const FETCH_TXS_BATCH_SIZE = 20;
 
+const UPDATE_NETWORK_IDS = [
+  0, // Cardano Mainnet
+  250, // Preprod Testnet
+  350, // Preview Testnet
+];
+
```
Comment (neuodev) [inline packages/yoroi-extension/app/api/ada/index.js]:
```suggestion
const UPDATE_NETWORK_IDS = [
  networks.CardanoMainnet.NetworkId,
  networks.CardanoPreprodTestnet.NetworkId,
  networks.CardanoPreviewTestnet.NetworkId
];

```

**Review by zuzunker — COMMENTED** [review]
/check

**Comment by yushih** [comment]
replaced by https://github.com/Emurgo/yoroi-frontend/pull/3608

**Pull request closed without merging**